packages/axios-extension/src/abap/adt-catalog/services/filestore-service.ts

```diff
@@ -0,0 +1,113 @@
+import { AdtService } from './adt-service';
+import type { AdtCategory, ArchiveFileNode, ArchiveFileNodeType, ArchiveFileContentType } from '../../types';
+import XmlParser from 'fast-xml-parser';
+import type { AdtFileNode } from 'abap/types/adt-internal-types';
+
+/**
+ * FileStoreService implements ADT requests to obtain the content
+ * of deployed archive.
+ *
+ * @class
```

Comment:
That's not needed but I also don't mind it.

Reply:
Removed while I was integrating changes based on @ullasholla  review.